package.json

```diff
@@ -61,10 +60,8 @@
     "tape": "^4.7.0"
   },
   "browser": {
-    "./lib/leveldown.js": false,
     "leveldown": false,
-    "leveldown/package": false,
-    "semver": false
+    "leveldown/package": false
```

Comment:
Are these really needed now? It seems to me we can remove the whole `browser` object unless there's something I'm missing.